### DISCUSSION TITLE
How do you install BS-RoFormer into UVR?

### DISCUSSION THREAD

**Discussion by ke7in-zz** [lucidrains/BS-RoFormer#31] [Q&A]
Installed BS-RoFormer via pip but not sure how to get it to show up in UVR. Thanks in advance!

**Comment by ghost** [comment]
BS/MelBand-Roformer integration into UVR is in beta phase currently.
Windows beta is available here: https://github.com/TRvlvr/model_repo/releases/download/uvr_update_patches/UVR_Patch_4_14_24_18_7_BETA_full_Roformer.exe
Stable release should be released soon.

**Reply by ghost** [reply]
Any idea if this beta works with MacOS and Apple Silcon? … or will that be sometime down the track?

**Reply by ghost** [reply]
There's an arm64 beta for MacOS that was shared in Audio Separation discord server:
https://cdn.discordapp.com/attachments/767947630403387393/1224923138479882310/Ultimate_Vocal_Remover_v5_6_MacOS_arm64.dmg?ex=667ae1d2&is=66799052&hm=02593e1ebe2a7e70b21e98d0ea6ec8ce2abb8d42c47502b60a48b85aae199c88&

For more support, we have to wait for next beta or stable version.


In the mean time, you can use ZFTurbo's script or the Google Colab version I've made around it:
https://github.com/jarredou/Music-Source-Separation-Training-Colab-Inference/
https://github.com/ZFTurbo/Music-Source-Separation-Training

**Reply by ghost** [reply]
> There's an arm64 beta for MacOS that was shared in Audio Separation discord server: https://cdn.discordapp.com/attachments/767947630403387393/1224923138479882310/Ultimate_Vocal_Remover_v5_6_MacOS_arm64.dmg?ex=667ae1d2&is=66799052&hm=02593e1ebe2a7e70b21e98d0ea6ec8ce2abb8d42c47502b60a48b85aae199c88&
> 
> For more support, we have to wait for next beta or stable version.
> 
> In the mean time, you can use ZFTurbo's script or the Google Colab version I've made around it: https://github.com/jarredou/Music-Source-Separation-Training-Colab-Inference/ https://github.com/ZFTurbo/Music-Source-Separation-Training

thanks so much! :)

**Comment by PurpBatBoi** [comment]
Is it already in UVR?

**Reply by ghost** [reply]
Still in beta phase, see my previous answers in this issue.

You can also join Audio Separation discord community https://discord.gg/ndC4UmPZwZ, UVR has a dedicated chan to follow upcoming updates, tips and custom models.

**Reply by nigz** [reply]
may i please get a new link to that discord?

**Comment by deton24** [comment]
Trying to keep that section updated with the newest UVR Roformer patches and instructions for various platforms. See [here](https://docs.google.com/document/d/17fjNvJzj8ZGSer7c7OFe_CNfUKbAxEh_OBv94ZdRG5c/edit?tab=t.0#heading=h.6y2plb943p9v).
